app/Http/Requests/SplitRequest.php

```diff
@@ -24,6 +24,7 @@ public function rules()
         return [
             'competition_end_date' => 'required|date|after:today',
             'leaderboard_msg_day' => 'required',
+            'rules' => 'required|url',
```

Comment:
Doesn't hurt to update it to `rules_url` and same in the form field attributes... Would be more explicit which isn't a bad thing ;)